ophyd/sim.py

```diff
@@ -418,39 +420,51 @@ def readback_func(x):
         super().__init__(name=name, parent=parent, labels=labels, kind=kind,
                          **kwargs)
         self.readback.name = self.name
+        self._events_per_move = events_per_move
+        self.egu = egu
+
+    def set(self, value: float) -> StatusBase:
```

Comment:
Small detail: we can be more precise here
```suggestion
    def set(self, value: float) -> MoveStatus:
```

Reply:
Interestingly I thought this wouldn't typecheck but turns out it does: https://mypy-play.net/?mypy=latest&python=3.7&gist=9b1b36bb342d46dacc8296f77746ee8e